tests/bootstrap.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
 define("MOCK_MINIMUM_VERSION", "0.40.0");
+define("MOCK_DOMAIN", getenv("STRIPE_MOCK_DOMAIN") ?: 'localhost');
```

Comment:
Two nits:
1. Could you rename the constant to `MOCK_HOST` (and the environment variable to `STRIPE_MOCK_HOST`)? I think host+port is a more common combination than domain+port.
2. Can you use double-quotes for the default value? We use double quotes everywhere else.

```suggestion
define("MOCK_HOST", getenv("STRIPE_MOCK_HOST") ?: "localhost");
```